app/controllers/application_controller.rb

```diff
@@ -293,7 +293,7 @@ def load_locales
   end
 
   # Sets the locale
-  def set_preferred_locale
+  def set_prefered_locale
```

Comment:
Assignment Branch Condition size for set_prefered_locale is too high. [18.14/15]